### PR TITLE
fix(alerts): Convert alert wizard to functional component

### PR DESCRIPTION
**Pull request by scttcper**
Remove analytic event nobody looks at `alert_wizard.option_viewed` we already had it disabled in amplitude

### CODE REVIEW

File: static/app/views/alerts/wizard/index.tsx [hunk 1]
```diff
@@ -34,47 +34,31 @@ type RouteParams = {
...
-  }
-
-  handleChangeAlertOption = (alertOption: AlertType) => {
-    this.setState({alertOption});
-    this.trackView(alertOption);
+function AlertWizard({
+  organization,
+  params,
+  location,
+  projectId: _projectId,
```
Comment (evanpurkhiser) [inline static/app/views/alerts/wizard/index.tsx:48]: Why does it need renamed?

Comment (scttcper) [reply static/app/views/alerts/wizard/index.tsx:48]: oh before it was because we had two of them. We attempt to get the project id from two places

Comment (evanpurkhiser) [reply static/app/views/alerts/wizard/index.tsx:48]: wait so do we still need this?

Comment (scttcper) [reply static/app/views/alerts/wizard/index.tsx]: yeah

**Review by evanpurkhiser — APPROVED** [review]

**Review by evanpurkhiser — APPROVED** [review]

**Pull request merged**